<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_0">
       <attribute name="title">
        <string>CPU Load</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Memory Load</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Checkpoint</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Execution Time</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Checkpoint Interval</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Checkpoint Directory</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Checkpoint Application</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Fault Rate</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Checkpoint Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="4" column="0">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="6" column="0">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Checkpoint Launch</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Close Checkpoint</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Coordinator Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Setting</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Optimal Interval</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Restart</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0" alignment="Qt::AlignTop">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Optimal Interval</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignTop">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
